pkg/workload/workload.go

```diff
@@ -51,6 +51,11 @@ func NewInfo(w *kueue.Workload) *Info {
 	}
 }
 
+func (i *Info) RebuildWorkload(wl *kueue.Workload) {
+	i.Obj = wl
```

Comment:
Field `ClusterQueue` should also updated here I think.

Reply:
done